_shared/project/.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     {% if job == 'coverage' %}
     needs: tests
     {% endif %}
-    runs-on: ubuntu-latest
+    runs-on: {{ cookiecutter["__" + job + "_runner_type"] }}
```

Comment:
This enables projects to customise the type of GitHub Actions runner used for each job by adding a setting like `"__tests_runner_type": "ubuntu-latest-64-cores"` to `cookiecutter.json`. This is to enable the faster runners that we've been using for LMS tests and lint